internal/rm/health.go

```diff
@@ -32,6 +32,7 @@ const (
 	// disabled entirely. If set, the envvar is treated as a comma-separated list of Xids to ignore. Note that
 	// this is in addition to the Application errors that are already ignored.
 	envDisableHealthChecks = "DP_DISABLE_HEALTHCHECKS"
+	envEnableHealthChecks  = "DP_ENABLE_HEALTHCHECKS"
```

Comment:
Can we rename this `DP_FORCE_HEALTHCHECKS` or `CRITICAL_XIDS` or `FATAL_XIDS` instead? (I would prefer the latter, but I could hear arguments for alternatives.

Reply:
`DP_DISABLE_HEALTHCHECKS` is the exact opposite of `DP_ENABLE_HEALTHCHECKS` and IMO the naming should reflect this. So I'd favour renaming DP_ENABLE_HEALTHCHECKS to DP_FORCE_ENABLE_HEALTHCHECKS, that way you get the "force" but it's still very clear it's the exact opposite of `DP_DISABLE_HEALTHCHECKS`.

Reply:
One question: What is the order of precedence of these? If `DP_DISABLE_HEALTHCHECKS=all|xids` does adding a specific XID to `DP_FORCE_ENABLE_HEALTHCHECKS` take precedence over this?

Then a comment on the naming. I don't think this envvar is the oposite of the existing one. While the existing one removes the predefined XIDs that we treat as fatal, this one adds ADDITIONAL ones.

Reply:
On balance I'd make `DP_ADDITIONAL_HEALTHCHECKS` /`DP_FORCE_ENABLE_HEALTHCHECKS` support `all`, and have it override `DP_DISABLE_HEALTHCHECKS`. This matters more if we choose a name with `FORCE` in it, but is probably worth doing regardless of name. This would complicate the code a little, but makes the external behavior to be what a reasonable user would expect.